usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,90 @@
 SAM_RECIPIENT_COLUMNS = {
-    "awardee_or_recipient_uniqu": "STRING",
-    "legal_business_name": "STRING",
-    "ultimate_parent_unique_ide": "STRING",
-    "ultimate_parent_legal_enti": "STRING",
-    "broker_duns_id": "INTEGER NOT NULL",
-    "update_date": "DATE NOT NULL",
-    "address_line_1": "STRING",
-    "address_line_2": "STRING",
-    "city": "STRING",
-    "congressional_district": "STRING",
-    "country_code": "STRING",
-    "state": "STRING",
-    "zip": "STRING",
-    "zip4": "STRING",
-    "business_types_codes": "ARRAY<STRING>",
-    "dba_name": "STRING",
-    "entity_structure": "STRING",
-    "uei": "STRING",
-    "ultimate_parent_uei": "STRING",
+    "awardee_or_recipient_uniqu": {"delta": "STRING", "postgres": "TEXT"},
+    "legal_business_name": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_unique_ide": {"delta": "STRING", "postgres": "TEXT"},
+    "broker_duns_id": {"delta": "STRING", "postgres": "TEXT"},
```

Comment:
This should be an `INTEGER` I believe

Reply:
Changed to Integer

Reply:
Still shows `STRING`

Reply:
In talking with Seth, I think given we're going to make this `int.sam_recipient`, and copy the ETL logic/procedure as it exists now in the pipeline to render USAS PG `raw.duns`, this could be left as `STRING` (since that's what it is in the USAS db). Eventually this delta data will be used to replace data in that pg table, so to minimize refactoring, I think keeping it as a STRING initially is correct. But we should take on tech-debt to do a deliberate refactor to `INTEGER`